cmd/root_test.go

```diff
@@ -4,7 +4,7 @@ import "testing"
 
 func TestRoot(t *testing.T) {
 	t.Run("Execute", func(t *testing.T) {
-		if err := Execute(); err != nil {
+		if err := Execute(nil); err != nil {
```

Comment:
nit: We should use a mock logger instead of nil here

Reply:
Mind taking a look at the mock I made?